docs/components/connectivity/wifi.md

```diff
@@ -1,4 +1,103 @@
 # WiFi
 
-!!! warning
-    This page is still in progress. You can check the description of blocks for help.
+## Overview
+
+The **WiFi** component can be used to manage the WiFi settings as well as to retrieve information about device's WiFi.
+
+## Methods
+
+### BSSID
+![](/assets/images/components/connectivity/wifi/m_bssid.png)
+
+\[None\]
+
+Returns the basic service set identifier (BSSID) of the current access point.
+
+### Disable
+![](/assets/images/components/connectivity/wifi/m_disable.png)
+
+\[ None \]  
+
+Disable the Wi-Fi
+
+### Enable
+![](/assets/images/components/connectivity/wifi/m_enable.png)
+
+\[ None \]
+
+Enable the Wi-Fi
+
+### Is 5GHz Supported
+![](/assets/images/components/connectivity/wifi/m_is-5ghz-supported.png)
+
+\[ None \]
+
+Return whether this adapter supports 5GHz band.
```

Comment:
Should be "Returns ..." (see above)

Also, how about providing some more info about the adapter and 5GHz band? You can place additional information in the `!!! info` dialog (see https://docs.kodular.io/components/utilities/color-utilities/#convert-int-to-hex)